core/build.gradle.kts

```
@@ -51,6 +51,8 @@ dependencies {
 
 tasks.test {
   maxParallelForks = Runtime.getRuntime().availableProcessors() * 2
+
+    systemProperty("testRuntimeClasspath", configurations.testRuntimeClasspath.get().asPath)
```

Another bit about matching existing formatting here

```suggestion
  systemProperty("testRuntimeClasspath", configurations.testRuntimeClasspath.get().asPath)
```

---

It's actually the line above that has an inconsistent indentation. Most of the file uses four spaces. I can change this anyway if you still want me to. :sweat_smile: